src/components/ReactComponent/blog/enhancements/diagram-renderer/DiagramEnhancer.tsx

```
@@ -0,0 +1,329 @@
+                  </kbd>
+                  <span>to exit</span>
+                </div>
+              </div>
+            </div>
+
+            {/* Modal Content */}
+            <div className="p-6 overflow-auto max-h-[calc(100vh-120px)] bg-theme-bg-card">
+              <div className="flex justify-center">
+                {fullscreenDiagram && (
+                  <div
+                    dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)